x/auth/client/cli/tx_multisign.go

```diff
@@ -152,7 +152,7 @@ func makeMultiSignCmd() func(cmd *cobra.Command, args []string) (err error) {
 				err = signing.VerifySignature(cmd.Context(), sig.PubKey, txSignerData, sig.Data,
 					txCfg.SignModeHandler(), txData)
 				if err != nil {
-					addr, _ := sdk.AccAddressFromHexUnsafe(sig.PubKey.Address().String())
+					addr, _ := sdk.AccAddressFromHex(sig.PubKey.Address().String())
```

Comment:
Wondering whether we should even enable this command in the first place.

Reply:
Yeah maybe we can remove it, but auth module will anyway fail any multisig tx as the decorator will reject txs with more than 1 signer